src/main/java/io/orangebeard/listener/OrangebeardTestSystemListener.java

```diff
@@ -375,18 +392,18 @@ private StartTestItem getStartTestItem(TestPage testPage) {
 
             if (pageData.getAttribute(WikiPageProperty.SUITES) != null) {
                 Set<Attribute> tags = convertAttributes(pageData.getAttribute(WikiPageProperty.SUITES));
-                testItem.attributes(tags);
+                testItem.attributes(new ArrayList<>(tags));
```

Comment:
Why make a set and then convert it to a list? I think attributes should be sets everywhere.

Reply:
but in listenerapi StartTest entity attributes is a list. so shall i change in listenerapi as well